python/Ganga/test/GPI/FileTests/TestMassStorageClient.py

```diff
@@ -30,13 +32,19 @@ def setUp(self):
         extra_opts=[('PollThread', 'autostart', 'False'),
                     ('Local', 'remove_workdir', 'False'),
                     ('TestingFramework', 'AutoCleanup', 'False'),
-                    ('Output', 'MassStorageFile', TestMassStorageClient.MassStorageTestConfig),
+                    ('Output', TestMassStorageClient.fileName, TestMassStorageClient.MassStorageTestConfig),
                     ('Output', 'FailJobIfNoOutputMatched', 'True')]
         super(TestMassStorageClient, self).setUp(extra_opts=extra_opts)
 
+    @classmethod
+    def getFileObject(cls):
+        """ Return an instance of the file we're wanting to test """
+        import Ganga.GPI as gpi
+        return getattr(gpi, cls.fileName)
```

Comment:
Moved to `fileClass`, lacking a proxy isn't a problem as the `stripProxy` is thankfully sensible. Just means that I now get the class name from the class vs the class from the class name.
I still think the plugin system can be redesigned to be better than it is so I'm reluctant to use it somewhere like the tests.